eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Binary/RoundTrip.hs

```diff
@@ -84,7 +84,6 @@ instance RuleListEra ConwayEra where
        , "GOV"
        , "LEDGER"
        , "LEDGERS"
-       , "MEMPOOL"
```

Comment:
I can't add `EraRuleFailure` type instance for `MEMPOOL` because that violates type injectivity.